main.c

```diff
@@ -171,6 +171,10 @@ static void process_share(enum nsaction *nsactions, const char *optarg)
 
 	int dirfd = AT_FDCWD;
 
+	// This is a special value we use to replace the path down below
+	// with the ns name when we're sharing with a directory of namespace files
+	char *share_with_dir = ((char *) -2);
```

Comment:
Why this couldn't be const?

Reply:
It's assigned back to `path` which is not const -- although I could have made this `char *const share_with_dir` but there aren't any real benefits in doing so.